lms/djangoapps/lti_provider/tests/test_views.py

```diff
@@ -45,14 +45,16 @@
 ALL_PARAMS = dict(LTI_DEFAULT_PARAMS.items() + COURSE_PARAMS.items())
 
 
-def build_launch_request():
+def build_launch_request(extra_post_data={}, param_to_delete=None):
```

Comment:
Usually not a good idea to have a mutable object as a default argument value

Reply:
+1